Mathlib/Algebra/Category/Grp/AB5.lean

```diff
@@ -0,0 +1,48 @@
+/-
+Copyright (c) 2023 David Kurniadi Angdinata. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Kurniadi Angdinata, Moritz Firsching, Nikolas Kuhn, Amelia Livingston
+-/
```

Comment:
Is this header correct?

Reply:
This is based on https://github.com/leanprover-community/mathlib4/pull/5597